public/data/salvator-del.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Brian Salvator",
+  "location": "Nairobi",
+  "bio": "Am a Fullstack web developer, infosec analyst and i love eveything about webdev and infosec.",
+  "avatar": "https://github.com/<your-github-username>.png",
```

```suggestion
  "avatar": "https://github.com/salvator-del.png",
```

add you github `username` so profile image can render.